docs/modules/user-guide/partials/assembly_migrating-to-devfile-v2.adoc

```diff
@@ -24,7 +24,7 @@ This guide explains how to migrate an existing devfile v1.x to a devfile v2.x.
 [role="_additional-resources"]
 .Additional resources
 
-For a deeper understanding of migrating devfiles, go to the GitHub issues on the following topics:
```

Comment:
Is there a reason why you deleted "For a deeper understanding of migrating devfiles"?

Reply:
Haha, I thought you suggested to make it cleaner and tighter to reuse the same formula for the additional resources section by  simply saying, "Go to the GitHub issue on the following topic..." 

Do you think it benefits from saying each time, "For a deeper understanding..."

Reply:
Resolving as I have made the suggested change.

Reply:
Yes, because the sentence you had was 
"Go to the following GitHub issue" without any context. But maybe the user doesn't actually need to go to these issues. It is not necessary, it is under the condition that the user wants to find out more. So instead of a plain imperative like in procedures, writers offer the additional resources to those interested, or who need them. Maybe they already have a thorough understanding of the topic, maybe they don't. If you use just an imperative, it seems like everyone needs to definitely visit the links. But then there is no context description as to why that would be the case.